content/docs/user-guide/dvc-files.md

```diff
@@ -118,14 +118,14 @@ and `dvc commit` commands, but not when a `.dvc` file is overwritten by
 
 ## `dvc.yaml` file
```

Comment:
just to reiterate, the whole document is overloaded, a lot of notes, a lot of ad-hoc structure - let's split this into a separate page right after we merge this

Reply:
Example removed. The note in the description now links to the section in dvc.yaml

Reply:
Agree. I mentioned this in https://github.com/iterative/dvc.org/pull/2052#issuecomment-751533344. I'm just not sure if the split can focus just on this new content or if we need to rethink the whole dvc.yaml ref vs. guide thing (#2059)...

Reply:
OK so after reading some other comments I started splitting this doc into several pages...